### DISCUSSION TITLE
'impact-engine' Command not Installed Following Getting Started Guide

### DISCUSSION THREAD

**Discussion by bvickers7** [Green-Software-Foundation/if#366] [Q&A] [answered]
Hello!

I'm trying to follow the [getting started guide](https://if.greensoftware.foundation/getting-started/), but am unable to run the `impact-engine` command after installing the `@grnsft/if@v0.1.3` package into a new yarn project. I'm just seeing:
`-bash: impact-engine: command not found`

I followed the instructions exactly so I am wondering if I am missing some prerequisite? I also noticed that the executable linked in [package.json](https://github.com/Green-Software-Foundation/if/blob/3b12d36c1f173050a30c90d5df3014833f334fb2/package.json#L10) is not in the source. Is that relevant?

I'm running
macOS Ventura 13.5
yarn 1.22.21
npm 9.6.7
node 18.7.1
@grnsft/if 0.1.3

Thanks in advance for the help!

**Comment by mads6879** [comment, marked as answer]
I had exact same issue when I tried to run impact-engine command after following the 'Getting started guide'.
Even though the yarn command `yarn add @grnsft/if` was successful and I could see the @grnsf package under node_modules, I was getting impact-engine not found error.
I then used the npm commands listed on https://github.com/Green-Software-Foundation/if
`npm install -g "@grnsft/if"`
This added around 85 packages and I was able to run impact-engine command.
Hope this helps.

**Reply by bvickers7** [reply]
That worked for me, thanks so much!
